src/interfaces/node.h

```diff
@@ -254,6 +255,10 @@ class Node
     using NotifyHeaderTipFn =
         std::function<void(bool initial_download, int height, int64_t block_time, double verification_progress)>;
     virtual std::unique_ptr<Handler> handleNotifyHeaderTip(NotifyHeaderTipFn fn) = 0;
+
+    //! Return pointer to internal chain interface, useful for testing.
+    //! Only works if node interface is being called in same process as implementation.
```

Comment:
I find comment fuzzy, do you mean only works if node interface is set by the process of the implementation ?